app/views/load_sample.py

```diff
@@ -4,7 +4,9 @@
 
 from app.auth import auth
 from app.controllers import action_controller
-from app.load_sample import load_sample
+from load_sample import load_sample
```

Comment:
If `load_sample` is the external package then this import line should sit above this local block:

```python
import stdlib_a
from stdlib_b import item

from third_party_package_a import another_item

from local_app import x
```

Reply:
didn't know that